ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -112,21 +124,21 @@ const BaseBulkContent = React.memo(({ actionDescription, details, project, name,
           <Table.Body>
             {optionalFields.map(field =>
               <TableRow key={field.header}>
-                <TableCell><BoldText>{field.header}</BoldText></TableCell>
+                <TitleTableCell><BoldText>{field.header}</BoldText></TitleTableCell>
                 <TableCell>{field.description}</TableCell>
               </TableRow>,
             )}
           </Table.Body>
         </StyledTable>
       </div>
-      {details && <div><br />{details}</div>}
-      <br />
     </Container>
+    {details && <div><br />{details}</div>}
```

Comment:
As I mentioned, the detail you added belongs in a field description so you should take this out